clarity_ext/context.py

```diff
@@ -196,6 +196,104 @@ def commit(self):
             obj.commit()
 
 
+class FakingEntityMonkey:
+    """
+    Prepare genologics.entities for an entity based mock by
```

Comment:
The docstring could be clearer. How should it be used and what's the expected outcome.